src/guides/v2.3/config-guide/deployment/pipeline/build-system.md

```diff
@@ -95,6 +95,13 @@ To set up the build system:
 
    See the [`.gitignore` reference]({{ page.baseurl }}/config-guide/prod/config-reference-gitignore.html) for more information.
 
+1. The build system should use either [default mode]({{ page.baseurl }}/config-guide/bootstrap/magento-modes.html#default-mode) or [developer mode]({{ page.baseurl }}/config-guide/bootstrap/magento-modes.html#developer-mode):
+
+   ```bash
+   bin/magento deploy:mode:set {mode}
+   ```
+   {mode} is required. It can be either default or developer.
```

Comment:
```suggestion
   **`{mode}`** is required. It can be either `default` or `developer`
```

Reply:
@ajithkumar-maragathavel 

The mentioned suggestion updated.